examples/react-native-todo-list/liveblocks.config.js

```diff
@@ -0,0 +1,13 @@
+import { createClient } from "@liveblocks/client";
+import { createRoomContext } from "@liveblocks/react";
+import { decode } from "base-64";
+
+const PUBLIC_KEY = "pk_YOUR_PUBLIC_KEY";
+
+const client = createClient({
+  publicApiKey: PUBLIC_KEY,
+  atobPolyfill: decode,
```

Comment:
@vinodsantharam As part of 0.17, I removed this API, and changed it to:

```suggestion
  polyfills: {
    atob: decode,
  },
```